jujugui/static/gui/src/app/components/deployment/deployment.js

```diff
@@ -30,7 +30,30 @@ YUI.add('deployment-component', function() {
       ecsCommit: React.PropTypes.func.isRequired,
       getUnplacedUnitCount: React.PropTypes.func.isRequired,
       jem: React.PropTypes.object.isRequired,
-      numberOfChanges: React.PropTypes.number.isRequired
+      numberOfChanges: React.PropTypes.number.isRequired,
+      users: React.PropTypes.object.isRequired
+    },
+
+    /**
+      Store information from portions of the deployment for use later down the
+      line.
+
+      @method setDeploymentInfo
+      @param key The key to store.
+      @param value The data to store.
+    */
+    setDeploymentInfo: function(key, value) {
+      this.deploymentStorage[key] = value;
+    },
+
+    /**
+      Retrieve stored deployment info.
+
+      @method getDeploymentInfo
+      @param key The key to retrieve.
+    */
+    getDeploymentInfo: function(key) {
```

Comment:
It will for when we delay adding templates to the final step, so I figured I'd add it now.